tox.ini

```diff
@@ -16,7 +16,7 @@ commands =
 passenv=HOME
 sitepackages = False
 commands =
-    flake8 --ignore=E501,W504 setup.py tests
+    flake8 setup.py tests
```

Comment:
Oh, here it is! No src/ directory.

Reply:
Yeah, turns out we need the ignores, didn't notice src was missing when I changed this